leakcanary-android-sample/build.gradle

```diff
@@ -87,15 +86,3 @@ android {
     }
   }
 }
-
-// Instrumentation test dependencies resolve to 27.1.1 so we align the sample.
-configurations.all {
-  resolutionStrategy {
-    eachDependency { details ->
-      // Force all of the primary support libraries to use the same version.
-      if (details.requested.group == 'com.android.support') {
-        details.useVersion "27.1.1"
-      }
-    }
-  }
-}
```

Comment:
Weren't used 🤷 Or at least I couldn't find a scenario where it was 🤷

Reply:
haha that was before we started supporting both Android X and the support library. The sample uses Android X though, so good one,